plugin/src/web.ts

```diff
@@ -9,10 +9,14 @@ import {
 } from './definitions';
 
 export class PortalsWeb extends WebPlugin implements PortalsPlugin {
-  async publish(_message: PortalMessage): Promise<void> { }
-  async subscribe<T = unknown>(_options: SubscribeOptions, _callback: SubscriptionCallback<T>): Promise<PortalSubscription> { return null as any; }
-  async unsubscribe(_options: PortalSubscription): Promise<void> { }
-  async echo(options: { value: string; }): Promise<{ value: string; }> {
-    return options;
+  async publish(_message: PortalMessage): Promise<void> {}
+
+  async subscribe<T = unknown>(_options: SubscribeOptions, _callback: SubscriptionCallback<T>): Promise<PortalSubscription> {
+    return {
+      subscriptionRef: -0,
+      topic: ""
+    }
```

Comment:
returning `null as any` is kind of a jerk move, we should return something here that isn't going make this a PITA.